frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx

```diff
@@ -136,6 +144,11 @@ const InstallCluster: React.FC<InstallClusterProps> = ({ match }) => {
       {mode === MODES.ATTACHED_DEVICES && (
         <CreateAttachedDevicesCluster match={match} mode={mode} />
       )}
+      <ExistingClusterModal
+        match={match}
+        clusterName={sc?.items?.[0]?.metadata?.name}
```

Comment:
clusterName should be filtered out based on the item which has status != ignored.
Check this:
https://github.com/openshift/console/blob/master/frontend/packages/ceph-storage-plugin/src/features.ts#L116..L119

Reply:
We want to block cluster creation , even if a single item is present.
And this condition is only valid to check  when someone fires from CLI a cluster when another cluster is already present. It seems an edge case and i dropped it but no harm in putting a check there !!